src/tools/ipv4-range-expander/ipv4-range-expander.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div>
+    <n-space item-style="flex:1 1 0">
+      <div>
+        <n-space item-style="flex:1 1 0">
+          <n-form-item label="Start address" v-bind="validationAttrsStart">
+            <n-input v-model:value="rawStartAddress" placeholder="Start IPv4 address..." />
+          </n-form-item>
+          <n-form-item label="End address" v-bind="validationAttrsEnd">
+            <n-input v-model:value="rawEndAddress" placeholder="End IPv4 address..." />
+          </n-form-item>
+        </n-space>
+
+        <n-table v-if="showResult" data-test-id="result">
```

Comment:
add a feedback when ip are valid but their is no result since range is invalid please

![image](https://user-images.githubusercontent.com/25065347/233137551-d3d76124-3435-4042-998d-445773b45447.png)